homeassistant/components/climate/daikin.py

```diff
@@ -75,9 +84,7 @@ def __init__(self, api):
         self._api = api
         self._force_refresh = False
         self._list = {
-            ATTR_OPERATION_MODE: list(
-                map(str.title, set(HA_STATE_TO_DAIKIN.values()))
-            ),
+            ATTR_OPERATION_MODE: list(DAIKIN_TO_HA_STATE),
```

Comment:
This should be:
```py
ATTR_OPERATION_MODE: list(HA_STATE_TO_DAIKIN),
```
We want to use the ha states so we use the keys of the `HA_STATE_TO_DAIKIN` dictionary.

Reply:
Can you check my lasts commit? I’m quite sure that I should use the other dictionary in that point. Today I’m not able to test since I’m travelling.

Reply:
I made the modification you suggested, but I'm not able to test it right now. I'm hoping that @mynameisdaniel32 can test it sooner than I can 😃

Reply:
I managed to run a quick test, and it seems to work good. As soon as I can I will run some further tests.